releases/release-8.5.5.md

```diff
@@ -0,0 +1,154 @@
+
+TiDB version: 8.5.5
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v8.5/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v8.5/production-deployment-using-tiup)
+
+## Features
+
+### Scalability
+
+* Placeholder for feature summary [#Issue-number](issue-link) @[Contributor-GitHub-ID](id-link) **tw@xxx** <!--1234-->
+
+    Provide a concise overview of what the feature is, the value it offers to users, and include a brief sentence on how to use it effectively. If there are any particularly important aspects of this feature, be sure to mention them as well.
+
+    For more information, see [Documentation](link).
+
+### Performance
+
+* Point-in-time recovery (PITR) supports recovery from compacted log backups for faster restores [#56522](https://github.com/pingcap/tidb/issues/56522) @[YuJuncen](https://github.com/YuJuncen) **tw@lilin90** <!--2001--> <!--TBD-->
+
+    Starting from v8.5.5, the log backup compaction feature provides offline compaction capabilities, converting unstructured log backup data into structured SST files. This results in the following improvements:
+
+    - **Improved recovery performance** - SST files can be more quickly imported into the cluster.
+    - **Reduced storage space consumption** - redundant data is removed during compaction.
+    - **Reduced impact on applications** - RPOs can be amintained with less frequent full snapshot-based backups.
+
+  For more information, see [documentation](/br/br-compact-log-backup.md).
+
+* Accelerated recovery of system tables from backups [#58757](https://github.com/pingcap/tidb/issues/58757) @[Leavrth](https://github.com/Leavrth) **tw@lilin90** <!--2109-->
+
+    When restoring system tables from a backup, BR now introduces a new `--fast-load-sys-tables` parameter to use physical restoration instead of logical restoration. This option completely overwrites/replaces the existing tables, instead of restoring into them, for faster restoration for large scale deployments.
+
+    For more information, see [Documentation](/br/br-snapshot-guide.md#restore-tables-in-the-mysql-schema).
+
+* Placeholder for feature summary [#Issue-number](issue-link) @[Contributor-GitHub-ID](id-link) **tw@xxx** <!--1234-->
+
+    Provide a concise overview of what the feature is, the value it offers to users, and include a brief sentence on how to use it effectively. If there are any particularly important aspects of this feature, be sure to mention them as well.
+
+    For more information, see [Documentation](link).
+
+### Reliability
+
+* Improved store scheduling in presence of network jitter [#9359](https://github.com/tikv/pd/issues/9359) @[okJiang](https://github.com/okJiang) **tw@qiancai** <!--2260-->
+
+    Provides a network status feedback mechanism to PD to avoid scheduling the leaders back to a problematic node (experiencing network jitter) after the leaders had been transferred off the node by TiKVs detection of the network jitter.
```

Comment:
```suggestion
    Provides a network status feedback mechanism to PD to avoid re-scheduling the leaders back to a problematic node (experiencing network jitter) after the leaders had been transferred off the node by TiKV raft mechanism. If the network continues to jitter, PD will actively evict leader from jittering node.
```